katas/content/marking_oracles/palindrome/index.md

```diff
@@ -0,0 +1,11 @@
+**Inputs:** 
+
+1. An array of $N$ qubits in an arbitrary state $\ket{x}$ (input register).
+2. A qubit in an arbitrary state $\ket{y}$ (target qubit).
```

Comment:
Let's use uniform description for inputs as well, borrowing from the first four tasks. This will make it easier for the learner to read them, just focus on the new information because everything else is the same